okhttp/src/jvmTest/java/okhttp3/ConnectionListenerTest.kt

```diff
@@ -52,11 +52,11 @@ open class ConnectionListenerTest {
   private val listener = RecordingConnectionListener()
   private val handshakeCertificates = localhost()
 
-  open val fastFallback: Boolean = true
+  open val fastFallback: Boolean get() = true
```

Comment:
What's the difference here?

Reply:
Avoids a potential problem accessing state from the superclass constructor.